ivy_tests/test_ivy/test_misc/test_mse_loss.py

```diff
@@ -0,0 +1,11 @@
+import ivy
+import ivy.functional.ivy.losses as l
+import numpy as np
+def mse_loss_test(x,y):
```

Comment:
The test should be located in ivy_tests/test_ivy/test_functional/test_nn/test_losses.py, and follow the same test style as the tests found in there. You can find more details about testing here https://unify.ai/docs/ivy/overview/deep_dive/ivy_tests.html.